src/plugins/intel_cpu/src/weights_cache.cpp

```diff
@@ -71,22 +71,23 @@ WeightsSharing::SharedMemory::Ptr WeightsSharing::get(const std::string& key) co
                                                 : std::unique_lock<std::mutex>(ptr->guard), ptr, newPtr);
 }
 
-NumaNodesWeights::NumaNodesWeights() {
-    for (auto numa_id : InferenceEngine::getAvailableNUMANodes())
-        _cache_map[numa_id] = std::make_shared<WeightsSharing>();
+SocketsWeights::SocketsWeights() {
+    int num_sockets = get_num_numa_nodes();
```

Comment:
`get_num_sockets` is now provided in PR18264. You can change to this interface.

Reply:
done.